test/reconfigurability.jl

```diff
@@ -30,3 +31,27 @@ using Test
     ssom_min = ssom(A, B, C, D)
     @test ssom_min >= 10 && ssom_min < 12
 end
+
+@testset "reconfigurability" begin
+    # # Refs
+    # [2] https://github.com/gramian/emgr
+    A = -Matrix(I, 4, 4)
+    B = [0.0 1.0 0.0 1.0]'
+    C = [0.0 0.0 1.0 1.0]
+
+    f(x, u, p, t) = A*x + B*u .+ p
+    g(x, u, p, t) = C*x
+
+    # System dimension
+    n = 4
+    m = 1
+    l = 1
+
+    x0 = [0.0, 0.0, 0.0, 0.0]
+    u0 = [0.0]
+
+    Wc = empirical_gramian(f, g, m, n, l; opt=:c, dt=0.01, tf=1.0, pr=zeros(4, 1), xs=x0, us=u0)
+    Wo = empirical_gramian(f, g, m, n, l; opt=:o, dt=0.01, tf=1.0, pr=zeros(4, 1), xs=x0, us=u0)
+    @show Wc Wo
+    @test Wc !== nothing && Wo !== nothing
```

Comment:
```suggestion
    @test Wc != nothing && Wo != nothing
```

[=== 와 ==의 차이](https://stackoverflow.com/questions/38601141/what-is-the-difference-between-and-comparison-operators-in-julia/38602012) 참고. 사실 현재 맥락에서는 !== 로 쓰나 != 로 쓰나 비슷할 것이긴 한데, 일단 != 로 하는게 좋을 것 같아요

Reply:
이거 하고 머지하시면 될듯!